site/content/contribute/server/feature-flags.md

```diff
@@ -26,13 +26,115 @@ Feature flags allow us to be more confident in shipping features continuously to
+ 2. Under the name of the split at the top, select the **Environment** that you want to target.
+     - See [Environments](#Environments) for a description of each.
+ 3. In the center of the screen, select **Add Rules**.
+ 4. Define your treatments:
+     - Recall that treatments are just the values your feature flag can have.
+     - Boolean flags should use the default "on" and "off" treatments.
+     - For non-boolean flags, change the treatments to be the possible values. E.g. for a split that specifies a plugin's version you could have three treatments of "1.0", "1.1" and "2.0".
+ 5. Down the page a bit, set the default rule to one of your treatments.
+     - This will set the flag for all servers in the environment while you don't have any other targeting rules configured.
+ 6. At the top right, click **Save changes** and confirm them on the next screen.
+
+That covers targeting an environment with a split and adding a default rule for all servers in that environment. The next section covers targeting specific servers.
+
+### Target a specific server
+
+The ID for a "server" as seen by Split is the Mattermost server's telemetry ID. To target a specific server you need that telemetry ID. The main way to get the telemetry ID from a server would be from the analytics but you can also manually grab the telemetry ID by one of the following methods:
+
+#### Get telemetry ID via inspecting browser traffic
+
+ 1. In Chrome (any browser with dev tools will work but instructions are specific to Chrome), open the developer console and switch to the Network tab.
+ 2. Type "config" into the filter for the network requests.
+ 3. Go to the server's URL (e.g. xxxxxx.test.mattermost.cloud).
+ 4. In the network tab, click on the request for `/api/v4/config/client?format=old`.
+ 5. Click on the "Response" tab and search for "TelemetryId".
+ 6. Copy the value of "TelemetryId".
+
+#### Get telemetry ID via curl and jq
+
+```bash
+$ curl -s https://<your-workspace-url>/api/v4/config/client?format=old | jq ".TelemetryId"
+```
+#### Target using the telemetry ID
+
+With the telemetry ID, go to your split and do the following:
+
+ 1. Find the section "Create whitelists" and click **Add whitelist**.
+ 2. Choose the treatment value you want the workspace to get.
+ 3. Under "Server" paste in your telemetry ID.
+ 4. At the top right, click **Save changes** and confirm them on the next screen.
+
+Now your workspace will have its feature flag set according to this rule, regardless of what the default for the split is set to.
+
+### Add or update a feature flag for community.mattermost.com
+
+Note that this will set the flag for community.mattermost.com, community-release.mattermost.com, and community-daily.mattermost.com. It is not possible to set a rule for only one.
+
+ 1. Open the split for your flag.
+ 2. At the top under the split name, select the "Cloud Staging" environment.
+ 3. Find "Set targeting rules" and click **Add rule**.
```

Comment:
Might want to comment that the targeting rule might already exist. Also isn't community the only thing on staging now? (i just set IC for the whole environment)

Reply:
Added a note about that. And yes right now the only thing hooked up to split in staging is community but in the future there could be other installations on staging using split too